.mergify.yml

```
@@ -0,0 +1,10 @@
+pull_request_rules:
+  - name: automerge to master with label automerge and branch protection passing
+    conditions:
+      - "#approved-reviews-by>1"
+      - base=master
+      - label=automerge
+    actions:
+      merge:
+        method: squash
+        strict: true
```

```suggestion
        strict: true
        commit_message: title+body
```

this takes in the title and description of prs. 

the only downside is all the check boxes that these templates have, but may be better to have a bot comment those check boxes

---

oh... taking in the checkboxes is not great

---

BTW I just copied the file as-is from cosmos-sdk

---

yes yes not a issue, I may make a pr in a bit to change something that may make mergify better. We did it on tendermint and find it way better